src/shim/shim-config.ts

```diff
@@ -138,15 +136,6 @@ class Config implements IConfig {
                 redis4: { enabled: true },
                 postgreSql: { enabled: true },
             },
-            logInstrumentationOptions: {
-                console: { enabled: false },
-                winston: { enabled: true },
-                bunyan: { enabled: true }
-            },
-            otlpTraceExporterConfig: {},
```

Comment:
Why removing OTLP configurations here? is it being initialized somewhere else not sure I follow

Reply:
Ah, this was a reaction to the API change in the distro, looks like I can modify the internal config of the distro from within `main.ts` so I'll just transition what used to be configurations passed to the distro via its API to modifying the internal config.